examples/java/src/main/java/org/apache/beam/examples/complete/TopWikipediaSessions.java

```diff
@@ -136,6 +141,20 @@ public void processElement(ProcessContext c, BoundedWindow window) {
     }
   }
 
+  static class ParseTableRowJson implements SerializableFunction<String, TableRow> {
+    private final ObjectMapper mapper =
+        new ObjectMapper().disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
```

Comment:
rather than reinvent the wheel, maybe use the built in JsonFactory class from the Google BigQuery libraries? It's used here:

https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java#L3113